packages/utils/src/estimate-gas-fees.ts

```diff
@@ -11,36 +12,45 @@ import { maxBigNumber } from './index';
  *   maxFeePerGas = baseFeePerGas + maxPriorityFeePerGas
  *   The baseFeePerGas depends on how full the previous blocks were.
  * - maxPriorityFeePerGas: The maximum priority fee per unit of gas for this transaction.
- * - gasPrice: Optional fallback: the gas price for this transaction.
  */
 async function estimateGasFees({
+  logger,
   provider,
   gasPriceMin,
 }: {
+  logger: LogTypes.ILogger;
   provider: providers.Provider | providers.JsonRpcProvider;
   gasPriceMin?: BigNumber;
 }): Promise<{
   maxFeePerGas?: BigNumber;
   maxPriorityFeePerGas?: BigNumber;
 }> {
-  const suggestedFee = await suggestFees(provider as providers.JsonRpcProvider);
+  try {
+    const suggestedFee = await suggestFees(provider as providers.JsonRpcProvider);
 
-  const baseFee = maxBigNumber(suggestedFee.baseFeeSuggestion, gasPriceMin || constants.Zero);
+    const baseFee = maxBigNumber(suggestedFee.baseFeeSuggestion, gasPriceMin || constants.Zero);
 
-  const maxPriorityFeePerGas = maxBigNumber(
-    suggestedFee.maxPriorityFeeSuggestions.urgent,
-    gasPriceMin || constants.Zero,
-  );
-  const maxFeePerGas = baseFee.add(maxPriorityFeePerGas);
+    const maxPriorityFeePerGas = maxBigNumber(
+      suggestedFee.maxPriorityFeeSuggestions.urgent,
+      gasPriceMin || constants.Zero,
+    );
+    const maxFeePerGas = baseFee.add(maxPriorityFeePerGas);
 
-  if (maxPriorityFeePerGas.eq(0) || maxFeePerGas.eq(0)) {
+    if (maxPriorityFeePerGas.eq(0) || maxFeePerGas.eq(0)) {
+      logger.warn(
+        `estimateGasFees: maxPriorityFeePerGas or maxFeePerGas too low (maxPriorityFeePerGas: ${maxPriorityFeePerGas.toString()} / maxFeePerGas: ${maxFeePerGas.toString()})`,
+      );
+      return {};
+    }
+
+    return {
+      maxPriorityFeePerGas,
+      maxFeePerGas,
+    };
+  } catch (e) {
+    logger.error(`estimateGasFees error: ${e}`);
```

Comment:
will this log correctly and not `[Object object]`?

Reply:
I've added a test, all good 👍